src/frontend/src/components/Charge/index.tsx

```diff
@@ -18,26 +18,35 @@ export default class Charge extends React.Component<Props> {
       expungement_result
     } = this.props.charge;
 
-    const knownDisposition = (disposition: any) => (
-      <>
+
+    const knownDisposition = (disposition: any, date: any) => {
+      let dispositionEvent;
+      if (disposition.ruling.toLowerCase().includes("convicted")) {
```

Comment:
I assume this is based on the logic in the backend. If the backend is ever changed then we have to remember to change the logic here. Do you think we should update the JSON response so that the charge object will have a field that states wether it is convicted or arrested so that we can remove this logic?

Reply:
Why don't we just display the disposition ruling as is (Convicted, Acquitted, Dismissed, etc)?

Reply:
@NickSchimek Yes, I like the idea of having this be simpler to read from the JSON response object. It should be part of #601 . I can add a note there.

@KentShikama The date only comes from the disposition if it was a conviction. If the disposition was an acquittal etc., the arrest date is what matters instead